### PR TITLE
[HOPSWORKS-2732] Add github actions for building documentation

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

File: .github/workflows/mkdocs-master.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: mkdocs
```
Comment (moritzmeister) [inline .github/workflows/mkdocs-master.yml:1]: You might need to give them unique names here, so we can distinguish them in the Github UI where the runs are shown.
```suggestion
name: mkdocs-master
```

File: .github/workflows/mkdocs-release.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: mkdocs
```
Comment (moritzmeister) [inline .github/workflows/mkdocs-release.yml:1]: Same as above.
```suggestion
name: mkdocs-release
```

**Review by moritzmeister — CHANGES_REQUESTED** [review]

File: .github/workflows/mkdocs-master.yml (hunk 1 above)
Comment (robzor92) [reply .github/workflows/mkdocs-master.yml]: I set unique names on all PRs

File: .github/workflows/mkdocs-release.yml (hunk 2 above)
Comment (robzor92) [reply .github/workflows/mkdocs-release.yml]: Done

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**